mkt/constants/iarc_mappings.py

```diff
@@ -327,16 +336,130 @@
         'USK_ShopStreamingService': '',
         'USK_Tabakkonsum': 'has_usk_tobacco',
     }
-
 }
 
-# Change {body: {'key': 'val'}} to {'val': 'key'}, combining v1 and v2 dicts.
-_REVERSE_DESCS_BY_BODY = [
-    {unicode(v): unicode(k) for k, v in body_mapping.items() if v}
-    for body, body_mapping in DESCS_V2.items() + DESCS.items()]
-REVERSE_DESCS = {}
-for mapping in _REVERSE_DESCS_BY_BODY:
-    REVERSE_DESCS.update(mapping)
+# Change {body: {'key': 'val'}} to {'val': 'key'}. Used to deserialize our
+# data back to IARC.
+REVERSE_DESCS_V2 = {}
+for mapping in [{unicode(v): unicode(k) for k, v in body_mapping.items() if v}
+                for body, body_mapping in DESCS_V2.items()]:
+    REVERSE_DESCS_V2.update(mapping)
+
+
+HUMAN_READABLE_DESCS_AND_INTERACTIVES = {
+    # Descriptors:
+    'has_classind_criminal_acts': _(u'Criminal Acts'),
```

Comment:
Might not worth the effort, most rating bodies display icons anyways. The rating bodies themselves make assumptions about the language based on the region so not sure we should try to spend time on this.

Before it was:

- English for ESRB for North America (I think frontend does translation for Spanish).
- English for PEGI for most of Europe.
- German for USK for Germany.
- Portuguese for CLASSIND for Brazil.